chapter-03/recipe-09/fortran-example/CMakeLists.txt

```diff
@@ -2,9 +2,10 @@
 cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 # project name and language
-project(recipe-09 LANGUAGES Fortran)
+# C language needed inside FindHDF5
+project(recipe-09 LANGUAGES Fortran C)
 
-find_package(HDF5 1.8 REQUIRED COMPONENTS Fortran)
+find_package(HDF5 1.8)
```

Comment:
Went to through too quick. The logic here is to build one or more Fortran executables that use HDF5, based on how HDF5 was compiled.
- `refregexample` **is always** compiled.
- `compound_fortran2003` is only compiled if the modern bindings are there.
- `ph5example` is only compiled if parallel I/O is in the HDF5 library.
The `find_package` will only look for C bindings, thus linking will always fail for these Fortran targets. I am modifying this recipe to get Travis green. On AppVeyor this one might have to be skipped completely.